Configuration/AlCa/python/autoCond.py

```diff
@@ -20,7 +20,7 @@
     # GlobalTag for Run1 data reprocessing
     'run1_data'         :   'GR_R_74_V12A::All',
     # GlobalTag for Run2 data reprocessing
-    'run2_data'         :   'GR_R_74_V13A::All',
+    'run2_data'         :   '74X_dataRun2_Prompt_v1::All',
```

Comment:
@mmusich Hi Marco, yes, I know Configuration/AlCa/python/autoCond_condDBv2.py is actually what is used, and it is set to 
  'run2_data' : '74X_dataRun2_v1',
already as what want.

The change in autoCond.py was historically left there from my first touch of it when I still didn't know this v1/v2 difference.  Because I also know this autoCond.py is not used, so i didn't spend time to change it back to GR_R_74_V13A::All', which i guess was also not valid anymore. Do you want me to change it back to 'GR_R_74_V13A::All' ?  Let me know, i can do that right away.